### PR TITLE
Docs Content: Tooltip

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
Updates Tooltip docs.

### CODE REVIEW

File: packages/docs/components/tooltip.md [hunk 1]
```diff
@@ -25,44 +25,54 @@ links:
...
+<Anatomy img="/images/anatomy-tooltip.svg" />
+
+## Behavior
+
 <Description>
 
-<span class="is-fpo">Descriptive content around **tooltip anatomy** should go here.</span>
+Tooltips activate when a cursor hovers over an element, or an element receives focus.
+
+Tooltips can be triggered both by the shared parent container or the paired sibling. If you are describing an element that can have focus, no more work is necessary.
```
Comment (arnoldsandoval-okta) [inline packages/docs/components/tooltip.md:36]: > Tooltips can be triggered both by the shared parent container or the paired sibling. If you are describing an element that can have focus, no more work is necessary.

this feels hard to grok without a visual. Suggest either rewording or providing a visual 

cc: @joejezowski-okta

File: packages/docs/components/tooltip.md [hunk 2]
```diff
@@ -224,66 +194,48 @@ The following positional classes are available:
...
-## Responsive considerations
+### Static
 
 <Description>
 
-With few exceptions, `:focus` and `:hover` interactions are typically unavailable on touchscreen devices, or trigger `:active` at the same time. (Android's <a href="https://material.io/design/interaction/gestures.html#types-of-gestures">long press action gesture</a> is one such exception.) This means tooltips on controls may be completely invisible for users of these devices.
+Tooltips should contain static content. Users don't expect, and are unlikely to notice, dynamic changes to Tooltip contents.
 
-Because no single touchscreen solution satisfies all tooltip usage types, consider fallback alternatives on a case-by-case basis.
+<strong>Exception:</strong> Tooltips may contain dynamic content if two requirements are met:
```
Comment (arnoldsandoval-okta) [inline packages/docs/components/tooltip.md:215]:
```suggestion
**Exception:** Tooltips may contain dynamic content if two requirements are met:
```

**Review by arnoldsandoval-okta — APPROVED** [review]

File: packages/docs/components/tooltip.md (hunk 1 above)
Comment (edburyenegren-okta) [reply packages/docs/components/tooltip.md:36]: This may be unnecessary here. Under the HTML tab, it'd be much easier to describe the setup.

Comment (edburyenegren-okta) [reply packages/docs/components/tooltip.md]: Reworded to feel more relevant to this section.

**Pull request merged**